Userland/Libraries/LibGfx/ImageFormats/TIFFLoader.cpp

```diff
@@ -42,9 +42,18 @@ class TIFFLoadingContext {
         return {};
     }
 
+    ErrorOr<void> ensure_baseline_tags_correctness() const
+    {
+        if (m_metadata.strip_offsets()->size() != m_metadata.strip_byte_counts()->size())
+            return Error::from_string_literal("TIFFImageDecoderPlugin: StripsOffset and StripByteCount have different sizes, aborting...");
```

Comment:
nit: get rid of the `, aborting...`, we don't do that anywhere else

(i realize the string is just moving around)